airflow/providers/databricks/operators/databricks.py

```diff
@@ -926,6 +926,7 @@ class DatabricksNotebookOperator(BaseOperator):
     """
 
     template_fields = ("notebook_params",)
+    CALLER = "DatabricksNotebookOperator"
```

Comment:
@pankajkoti , do you think its better to make caller set to "Unknown" by default? or the current one is fine?

Reply:
This is in reference to https://github.com/apache/airflow/blob/eb48911817b100615a1c138de3a71fbbf4bd580a/airflow/providers/databricks/hooks/databricks_base.py#L106

Reply:
I think it's because `BaseDatabricksHook` is the base class of all the databricks hooks. May I know why do we want to make this change?

Reply:
There was no caller actually being passed to the base hook from `DatabricksNotebookOperator`. So, I added it. Just wanted to confirm what value I should keep in the `DatabricksNotebookOperator` class.